meeting_minutes/2021-08-31

```diff
@@ -0,0 +1,55 @@
+# working group meeting on 2021-08-31
+
+The agenda is in the [Kanban board](https://github.com/llvm/llvm-iwg/projects/1)
+
+## Participants
+* Christian Kühnel
+* Anja Gerbes
+* Wei Wu
+* Mishal Shah
+* Anton Korobeynikov
+* Tom Stellard
+
+## Document which sub-projects are using Phabricator and PullRequests #21
+
+* the first phabricator entry is ready 
+
+## Add Infrastructure Admin description #41
+
+* Foundation wants to hire a contractor (not an individual)
+* [proposal on a process for hirning a contractor](https://docs.google.com/document/d/1g9C4pEvTiq65vhKciydGvjXW3kBYuxaPGtt7D4Bz0E4/)
+  * meant to start the discussion on outsourcing
+  * we might want to convert this into markdown and store it in the repo
+
+## proposal on IWG priorities #52
+
+*  to be merged
+
+## Future of CI and Pull Requests
+
+* Wei Wu has experience with setting up runners and is willing to help
+* GitHub Pull Request disabled as they share the namespace with github issues
+  so the numbers would get used for PRs. However we want to reserve the numbers
+  for the migration of the bugzilla bugs and preserve their IDs.
+  * see also [our last meeting](https://github.com/llvm/llvm-iwg/blob/main/meeting_minutes/2021-08-17.md#update-on-bugzilla-migration)
+  * Timeline for bugzilla/issue migration: next few months
+* We need to differentiate two topics, to be discussed separately:
+  * What is actually technically needed for Pull Requests? (checking CLA, licences, issue IDs,...)
+  * What are users experience during the review process?
+* Open points: 
+  * How do we convince the community? Why do we thing this is the right tool?
+  * What is the biggest push-back from the community?
+    * Review interface is not as feature-rich as Phabricator.
+    * Reviewing stacked patches is more ugly.
```

Comment:
"more ugly" is an under-statement: you could say "not supported".

Something to add to the list here is: "lack of Herald rules". When we had a round-table on PullRequest at the dev mtg in 2019 this was considered a hard blocker by a few. After that I searched online but couldn't find any alternative readily usable for GitHub. This is something that might be possible to implement ourselves with bots polling GitHub and managing the subscription, but this needs "work".